src/assets/scripts/client/aircraft/Pilot/Pilot.js

```diff
@@ -528,17 +528,27 @@ export default class Pilot {
     *
     * @for Pilot
     * @method exitHold
+    * @param fixName {string} name of the fix at which the hold should be canceled (optional)
+    * @return {array} [success of operation, readback]
     */
-    exitHold() {
-        const currentWaypoint = this._fms.currentWaypoint;
+    exitHold(fixName) {
+        let waypoint = this._fms.currentWaypoint;
 
-        if (!currentWaypoint.isHoldWaypoint) {
+        if (fixName) {
+            waypoint = this._fms.findWaypoint(fixName);
+
+            if (!waypoint) {
```

Comment:
let's move this `if` outside of the current `if`.  that'll give us a little extra defense

Reply:
done